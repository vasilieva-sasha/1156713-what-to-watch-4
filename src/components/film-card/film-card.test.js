import React from "react";
import renderer from "react-test-renderer";
import FilmCard from "./film-card";

const film = {
  title: ``,
  poster: ``
};

describe(`FilmCardComponent`, () => {
  it(`FilmCard correct render`, () => {
    const tree = renderer.create(
        <FilmCard film={film} onArticleHover={() => {}} onTitleClick={() => {}} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
