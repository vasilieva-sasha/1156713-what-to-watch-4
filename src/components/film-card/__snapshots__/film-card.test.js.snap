// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmCardComponent FilmCard correct render 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
    onClick={[Function]}
  >
    <video
      poster="img/.jpg"
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
    />
  </div>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      
    </a>
  </h3>
</article>
`;
